submitit/helpers.py

```diff
@@ -220,8 +220,31 @@ def __exit__(self, *args):
         os.chdir(self.original_dir)
 
 
+def _default_monitor_callable(
```

Comment:
I think it could be nice if `as_completed` could also accept such a callback and regurlarly update us about the progression. 
I usually end up implementing this myself in all my project.

Note: left for another PR